operator/pkg/model/translation/cec_translator.go

```diff
@@ -59,14 +60,15 @@ type cecTranslator struct {
 }
 
 // NewCECTranslator returns a new translator
-func NewCECTranslator(secretsNamespace string, useProxyProtocol bool, useAppProtocol bool, hostNameSuffixMatch bool, idleTimeoutSeconds int,
+func NewCECTranslator(secretsNamespace string, useProxyProtocol bool, useAppProtocol bool, useAlpn bool, hostNameSuffixMatch bool, idleTimeoutSeconds int,
```

Comment:
Originally, we have any a few params, but this list is getting bigger and bigger, wdyt about changing this list to a struct ? We can even extend a little bit more with functional extension pattern (e.g. WithUseAlpn)?

Reply:
That would work. Should I only do it for ALPN?

Reply:
if possible, you can do it in the first commit, the second one will be to support alpn.

Reply:
ah ignore my above comment, doing it for ALPN is fine, I can do refactor separately.

Reply:
@sayboras please see if I understood it correctly. I made it togglable like `WithUseAlpn(useAlpn bool)`.

Reply:
yes, it should be good enough :heavy_check_mark: